frontend/src/libs/components/customer-card/styles.module.scss

```diff
@@ -0,0 +1,25 @@
+@import "src/assets/css/vars.scss";
+
+.card {
+  max-width: 540px;
+  padding: 27px 89px 39px 222px;
+  background-color: $white;
+  background-image: url("../../../assets/img/welcome-page/customer.png");
+  background-repeat: no-repeat;
+  background-size: 229px 229px;
```

Comment:
Please use Image element inside your JSX instead of playing with background-size

Reply:
it's better to use <img/> in JSX component?
First I used <img>, but really this picture is non content image (in my opinion), so I moved it to css file

Reply:
move images to JSX component